systems/subscriber/sim-manager/pkg/server/sim_manager.go

```diff
@@ -229,6 +236,28 @@ func (s *SimManagerServer) AllocateSim(ctx context.Context, req *pb.AllocateSimR
 	if err != nil {
 		log.Errorf("Failed to publish message %+v with key %+v. Errors %s", req, route, err.Error())
 	}
+	netInfo, err := s.networkClient.GetNetworkInfo(remoteSubResp.Subscriber.NetworkId, remoteSubResp.Subscriber.OrgId)
+	if err != nil {
+		return nil, status.Errorf(codes.NotFound, "network not found for that org %s", err.Error())
+	}
+
+	emailBody, err := utils.GenerateEmailBody(netInfo.Name, remoteSubResp.Subscriber.FirstName, poolSim.QrCode)
+	if err != nil {
+		return nil, err
+	}
+	if poolSim.QrCode != "" {
```

Comment:
Simple question for me: Why do we send email for sim allocation when QrCode is empty?